package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "conventional-pr-labels",
+  "module": "src/index.ts",
+  "type": "module",
+  "scripts": {
+    "build": "bun build src/index.ts --outdir ./dist",
+    "test": "bun test",
+    "test:watch": "bun test --watch"
+  },
+  "devDependencies": {
+    "bun-types": "latest"
+  },
+  "peerDependencies": {
+    "typescript": "^5.0.0"
+  },
```

Is there a reason for adding it as a peer dependency? This module will never been used as a dependency for any other Node projects.

```suggestion
  "devDependencies": {
    "bun-types": "latest",
    "typescript": "^5.2.0"
  },
```